backend/app/Http/Controllers/User/ShowIndividualUserController.php

```diff
@@ -28,7 +40,127 @@ public function __construct(Gate $gate)
+            ->first();
+
+        $userAreas = Area::whereIn('id', [$user->birth_area_id, $user->live_area_id])->get();
+        // 在住と出身が同じだった場合、返り値1つなので
+        /** @extends  */
+        $userBirthArea = $userAreas->first(fn (Area $area) => $area->id === $user->birth_area_id);
+        $userLiveArea = $userAreas->first(fn (Area $area) => $area->id === $user->live_area_id);
+        // userにbirthとliveメソッドを追加
+        $user->birth_area = $userBirthArea->prefecture_code->label() . $userBirthArea->municipality;
+        $user->live_area = $userLiveArea->prefecture_code->label() . $userLiveArea->municipality;
+
+        $links = DB::table('user_links')
+            ->where('user_id', $user_id)
+            ->get();
+
+        $projects = DB::table('project_belongeds')
+            ->where('user_id', $user_id)
+            ->join('projects', 'project_belongeds.project_id', '=', 'projects.id')
+            ->get();
+
+        $events = UserTimeline::where('user_id', $user_id)
+            ->orderBy('start_date', 'asc')
+            ->get();
+
+        $relatedUsers = DB::table('user_infos')
+            ->where('grade', $user->grade)
+            ->join('users', 'user_infos.user_id', '=', 'users.id')
+            ->leftJoin('u_u_majors', 'user_infos.u_u_major_id', '=', 'u_u_majors.id')
+            ->select('users.name', 'users.profile_photo_path', 'user_infos.user_id', 'user_infos.grade', 'user_infos.u_u_major_id', 'user_infos.university_meta', 'user_infos.company_meta', 'u_u_majors.name as uu_major', 'u_u_majors.faculty_id as uu_faculty')
+            ->inRandomOrder()
+            ->limit(2)
+            ->get();
+
+        /**
+         * enumを用いて学部・学年・性別・国・都道府県・タイムラインのジャンル・関連ユーザを取得
+         */
+        $user->uu_faculty = $user->uu_faculty == null ? "" : UUFaculty::from($user->uu_faculty)->label();
+        $user->grade = Grade::from($user->grade)->label();
+        $user->gender = Gender::from($user->gender)->label();
+        foreach ($events as $event) {
+            $event->genreName = $event->genre->label();
+        }
+
+        /**
+         * userにprofessionメソッドを追加
+         * 社会人→会社名＋役職
+         * 他大生→学名＋学部＋学科
+         * 宇大生→学部＋学科
+         * (宇大生から社会人になった場合、宇大のデータが残っている可能性が否めないので、それ防止のために先に社会人から条件分岐させている)
+         */
+        if ($user->company_meta) {
+            $company_meta = json_decode($user->company_meta);
+            $user->profession = $company_meta->company_name . " " . $company_meta->position;
+        } else if ($user->university_meta) {
+            $university_meta = json_decode($user->university_meta);
+            $user->profession = $university_meta->university . " " . $university_meta->faculty . " " . $university_meta->major;
+        } else if ($user->uu_faculty) {
+            $user->profession = $user->uu_faculty . " " . $user->uu_major;
+        } else {
+            //この場合は存在しないはずだがエラーになるのは困るので空文字
+            $user->profession = '';
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Access to an undefined property App\Models\UserInfo::$profession.